Classes/Utility/ColumnLayoutUtility.php

```diff
@@ -112,4 +112,51 @@ public static function hydrateLayoutConfigFlexFormData($flexFormData): array
             }, $sheet['lDEF']);
         }, $dataStructure['data']);
     }
+
+    /**
+     * Get available and configured additional column layout classes.
+     *
+     * @param int $pageUid
+     * @return array
+     */
+    public function getAvailableLayouts(int $pageUid): array
+    {
+        $layouts = [];
+
+        // Check if the layouts are extended by ext_tables
+        if (isset($GLOBALS['TYPO3_CONF_VARS']['EXT']['column_layout']['additionalLayouts'])
+            && is_array($GLOBALS['TYPO3_CONF_VARS']['EXT']['column_layout']['additionalLayouts'])
+        ) {
+            $layouts = $GLOBALS['TYPO3_CONF_VARS']['EXT']['column_layout']['additionalLayouts'];
+        }
+
+        // Add TsConfig values
+        foreach ($this->getLayoutsFromTsConfig($pageUid) as $templateKey => $title) {
+            if (GeneralUtility::isFirstPartOfStr($title, '--div--')) {
+                $optGroupParts = GeneralUtility::trimExplode(',', $title, true, 2);
+                $title = $optGroupParts[1];
+                $templateKey = $optGroupParts[0];
```

Comment:
So if the title contains a `--div--`, the original `$templateKey` is ignored but instead the one from the the title?
Is there a reason for that behaviour? If so, could you add an inline comment explaining it?

Reply:
You still don't want to change this?